frontend/test/pytest/test_jvpvjp.py

```diff
@@ -832,5 +832,111 @@ def J_workflow():
         assert_allclose(r_j, r_c)
 
 
```

Comment:
So there were no test failing because of tuple/list args?

Reply:
I removed the check that requires the PyTree structures be the same (like the one performed in JAX). I don't think it makes sense to have it here since Catalyst allows a mixture of differentiable and non-differentiable parameters, so it's legal for the trees of the primal and tangent arguments to be different. Also a user might find it frustrating that `PyTree((*, *))` and `PyTree([*, *])` are incompatible! :)

Reply:
Yes I forgot Jax has no `argnum` for JVPs/VJPs